x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/deprecations_count_checkpoint/deprecations_count_checkpoint.tsx

```diff
@@ -51,26 +47,20 @@ const i18nTexts = {
   ),
 };
 
-const getPreviousCheckpointDate = () => {
-  const storedValue = moment(localStorage.get(LS_SETTING_ID));
-
-  if (storedValue.isValid()) {
-    return storedValue.toISOString();
-  }
-
-  const now = moment().toISOString();
-  localStorage.set(LS_SETTING_ID, now);
+interface Props {
+  lastCheckpoint: string;
+  resetLastCheckpoint: (value: string) => void;
```

Comment:
Nit: "last" doesn't seem to provide much information in these names, since we are only ever dealing with a single checkpoint.